ts/pages/staking/wizard/wizard.tsx

```diff
@@ -56,33 +51,29 @@ const Container = styled.div`
 export const StakingWizard: React.FC<StakingWizardProps> = props => {
     // If coming from the market maker page, poolId will be provided
     const { poolId } = useQuery<{ poolId: string | undefined }>();
-    const { providerState } = props;
+    const { connector, account, chainId } = useWeb3React<Web3Wrapper>();
 
     const dispatch = useDispatch();
     const dispatcher = new Dispatcher(dispatch);
 
-    const networkId = useSelector((state: State) => state.networkId);
-    const apiClient = useAPIClient(networkId);
+    const apiClient = useAPIClient(chainId);
 
     const [stakingPools, setStakingPools] = useState<PoolWithStats[] | undefined>(undefined);
     const [selectedStakingPools, setSelectedStakingPools] = React.useState<UserStakingChoice[] | undefined>(undefined);
     const [currentEpochStats, setCurrentEpochStats] = useState<Epoch | undefined>(undefined);
     const [nextEpochStats, setNextEpochStats] = useState<Epoch | undefined>(undefined);
     const [allTimeStats, setAllTimeStats] = useState<AllTimeStats | undefined>(undefined);
+    const [zrxBalance, setZrxBalance] = useState<BigNumber | undefined>(undefined);
+    const [zrxBalanceBaseUnitAmount, setZrxBalanceBaseUnitAmount] = useState<BigNumber | undefined>(undefined);
+    const [zrxAllowance, setZrxAllowance] = useState<BigNumber | undefined>(undefined);
```

Comment:
We need to pull this from redux instead, otherwise we're duplicating the source of truth and it can get out of sync. (e.g. onUpdateAccountBalances gets called and the zrxBalance balance here won't get updated.